website/example.css

```diff
@@ -0,0 +1,163 @@
+body, html {
+    height: 100%;
+    padding: 0;
+    margin: 0;
+    font-family: Avenir Next;
+}
+
+* {
+    padding: 0;
+    margin: 0;
+    border: none;
+}
+
+*:focus {
+    outline: none;
+}
+
+a {
+    text-decoration: none;
+    color: black;
+}
+
+/* ------------------------------------------------------------------------------------------------------------------------ */
```

Comment:
I believe you are going for a CSS reset here, but using the global `*` will break a lot of different things, and globally kills any new elements going forward. as the last design lasted almost four years this is probably a bad idea. I'd recommend something like the [meyerweb](https://meyerweb.com/eric/tools/css/reset/) reset here. It does most all of this without the break all.

Additionally you're dealing with a lot of border and padding headaches you could avoid by using the [border-box](https://css-tricks.com/box-sizing/#article-header-id-6) styling reset as well. Please read the attached article and I can answer any questions you may have going forward.